package.json

```diff
@@ -13,19 +14,40 @@
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
     "express": "~4.16.1",
-    "http-errors": "~1.6.3",
     "jade": "~1.11.0",
     "memcached": "^2.2.2",
     "morgan": "~1.9.1",
     "request": "^2.88.0",
     "validator": "^12.2.0"
   },
   "devDependencies": {
+    "@types/chai": "4.2.10",
+    "@types/mocha": "7.0.1",
+    "@types/node": "13.7.7",
     "chai": "4.2.0",
+    "eslint": "6.8.0",
+    "eslint-config-bloq": "2.3.0",
+    "eslint-config-prettier": "6.10.0",
+    "eslint-config-standard": "14.1.0",
+    "eslint-plugin-import": "2.20.1",
+    "eslint-plugin-jsdoc": "21.0.0",
+    "eslint-plugin-mocha": "6.3.0",
+    "eslint-plugin-node": "11.0.0",
+    "eslint-plugin-prefer-arrow": "1.1.7",
+    "eslint-plugin-prettier": "3.1.2",
+    "eslint-plugin-promise": "4.2.1",
+    "eslint-plugin-require-path-exists": "1.1.9",
+    "eslint-plugin-standard": "4.0.1",
+    "husky": "4.2.3",
+    "lint-staged": "10.0.8",
+    "prettier": "1.19.1",
     "chai-as-promised": "7.1.1",
     "mocha": "7.0.1",
     "nyc": "15.0.0",
     "randomstring": "1.1.5",
     "request-promise-native": "1.0.8"
+  },
+  "lint-staged": {
```

Comment:
We should configure lint-staged in a different file, as in all the other `BloqCloud` repositories. See https://github.com/bloqpriv/cloud-billing/blob/develop/.lintstagedrc.json

Reply:
Like this. We should do the same on `eth-connect`. More reasons towards mono-repos.